conan_test_docker/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM conanio/gcc5
```

Comment:
copy of conan_tests + Docker package installation and system docker enable.

More information  how to install on linux: https://docs.docker.com/engine/install/ubuntu/